authorization-service/config.ts

```diff
@@ -0,0 +1,7 @@
+import * as dotenv from 'dotenv';
+
+dotenv.config({ path: '.env.default' });
+
+export default {
+  petr150894: process.env.petr150894
```

Comment:
Actually it's extra work if you don't use it anywhere else, serverless-dotenv-plugin upload env variables automatically from .env file